web/sites/default/services.yml

```diff
@@ -81,7 +81,7 @@ parameters:
     #
     # Enabling auto-reload is not recommended in production environments.
     # @default null
-    auto_reload: null
+    auto_reload: true
```

Comment:
Do not change the services.yml. Use local overrides to keep these changes only for the local environments - https://docs.google.com/document/d/14R23U9gWW9Q82pPUWLGh9REBk-5BI6LtDjxNQoDonZg/edit?usp=sharing